### ISSUE TITLE
When we are importing the API with --params file flag, all the gateways are getting checked in the second environment, even the exported API zip file has only one Gateway environment.

### ISSUE THREAD

**Issue by riyazathali** [wso2/product-apim-tooling#497]
**Description:**
When we are importing the API with, all the gateways are getting checked in the second environment, even the exported API zip file has only one Gateway environment.
This issue is happening when we are using the --params flag.

**Affected Product Version:**
APIM 2.6.0, apimcli-2.0.9-macosx-x64.tar, api-import-export-2.6.0-v17.war

**Steps to reproduce:**
1. Configure 2 gateway environments(internal and external) in api-manager.xml file of the publisher from both 2 different environments(dev and prod)
2. Check the one gateway environment in the first environment(dev) and export the API.
`./apimcli export-api -n SampleAPI -v 1.0.0 -r admin -e dev -k`
3. You can see that the following in the api.json in the exported zip file. (One environment is there)
```
"environments": [
    "Internal Gateway"
  ]
```
4. Import the API to the second environment(prod) with the --params flag (api.yaml file) by using the following command. 
`./apimcli import-api -f dev/SampleAPI_1.0.0.zip -e prod --params ../api.yaml -k`
5. The above api.yaml file has the following content.
```
environments:
    - name: prod
      endpoints:
          production:
              url: http://updatedURL
          sandbox:
              url: http://updatedURL
```
6. We can see that both the gateways((Internal and External)) are checked in the second environment.

**Comment by wasuradananjith** [comment]
This is already fixed in the master branch via [1]. Hence can close the issue.

[1] https://github.com/wso2/product-apim-tooling/pull/303

**Comment by npamudika** [comment]
The fix will be available in the APIM CLI 2.0.10 release.
